cli/src/pcluster/api/controllers/common.py

```diff
@@ -29,22 +29,26 @@
 )
 from pcluster.aws.common import BadRequestError, LimitExceededError, StackNotFoundError, get_region
 from pcluster.config.common import AllValidatorsSuppressor, TypeMatchValidatorsSuppressor, ValidatorSuppressor
-from pcluster.constants import SUPPORTED_REGIONS, UNSUPPORTED_OPERATIONS_MAP, Operation
+from pcluster.constants import UNSUPPORTED_OPERATIONS_MAP, Operation
 from pcluster.models.cluster import Cluster
 from pcluster.models.common import BadRequest, Conflict, LimitExceeded, NotFound, parse_config
-from pcluster.utils import get_installed_version, to_utc_datetime
+from pcluster.utils import get_installed_version, retrieve_supported_regions, to_utc_datetime
 
 LOGGER = logging.getLogger(__name__)
 
 
 def _set_region(region):
     if not region:
         raise BadRequestException("region needs to be set")
-    if region not in SUPPORTED_REGIONS:
```

Comment:
Why not just do: `if region not in retrieve_supported_regions():` here and then keep the rest of the old logic the same?